framework/src/media/MediaPlayerImpl.cpp

```diff
@@ -378,15 +378,15 @@ int MediaPlayerImpl::getVolume()
 	return ret;
 }
 
-void MediaPlayerImpl::getVolumePlayer(int &ret)
+void MediaPlayerImpl::getVolumePlayer(uint8_t &ret)
 {
 	medvdbg("MediaPlayer Worker : getVolume\n");
-
-	ret = get_output_audio_volume();
+	//ToDo: Add error handling code
+	get_output_audio_volume(&ret);
```

Comment:
ret or &ret
is it correct?

Reply:
&ret is correct. (Operations are checked.)
ret causes compile errors.